lib/internal/streams/readable.js

```diff
@@ -379,7 +376,7 @@ function howMuchToRead(n, state) {
     return 0;
   if (state.objectMode)
     return 1;
-  if (NumberIsNaN(n)) {
+  if (Number.isNaN(n)) {
```

Comment:
Do you have any evidence that "static" primordials can lead to performance issues? I'm okay with reverting the prototype ones but this is too much imo.

Reply:
I'm asking because `NumberIsNan` is literally `Number.isNaN` (safe function identity)

Reply:
I don't. The approach I used is to remove all of them from hot paths, literally with `%s/primordial/replacement/g`. If you want I can try adding those back in.

I'm 99.9% sure on the ones from `*Prototype*` are the major culprit (functions and arrays specifically).